test/verify/check-system-info

```diff
@@ -288,8 +288,15 @@ class TestSystemInfo(PackageCase):
         b = self.browser
 
         if m.image.startswith("debian") or m.image.startswith("ubuntu") or m.image == "arch":
-            # timesyncd is default
-            pass
+            if m.execute("type chronyc || true").strip() != "":
+                # chronyd is default, install timesyncd
+                self.addPackageSet("timesyncd")
+                self.enableRepo()
+                m.execute("apt-get update; apt-get install -y systemd-timesyncd")
```

Comment:
I suppose that uninstalls chrony due to the package conflict?

Reply:
Yes, and also freeipa-client.